spec/controllers/users/two_factor_authentication_controller_spec.rb

```diff
@@ -594,6 +596,70 @@ def index
           )
           expect(response).to redirect_to authentication_methods_setup_url
         end
+        expect(@analytics).to have_logged_event(
+          'Rate Limit Reached',
+          {
```

Comment:
Curly braces technically optional if we wanted to omit?

Reply:
Ah right, thanks, amended into 1c00b914ea